css/orderpage.css

```diff
@@ -0,0 +1,184 @@
+
+ :root {
+    --grey-dark: #848993;
+    --grey-darker: #4D5057;
+    --grey-light: #E5E5E5;
+    --orange-dark: #E0633A;
+    --orange-light: #FCEFEB;
+    --white: #fff;
+}
+main {
+    display:flex;
+    margin-left:7%;
+    min-height: 80vh;
+}
+main > section {
+    width: 50%;
+}
+.form-container {
+    margin: auto 7% auto 0; 
+}
+
+.image-container {
+    text-align: right;
+}
+.form-container h1 {
+    font-size: 3rem;
+    font-weight: 500;
+    color:var(--orange-dark);
+    padding-bottom:1rem;
+}
+form {
+    display: flex;
+    flex-direction: column;
+    color:var(--grey-darker);
+}
+label {
+    margin-bottom: 0.4rem;
+}
+form input,
+form select {
+    margin-bottom: 15px;
+    padding:5px;
+    border: 1px solid var(--grey-darker);
+    border-radius: 4px;
+}
+form select {
+    background-color: var(--white);
+}
+form button {
+    width: 50%;
+} 
+.requiredstar {
+    color: var(--orange-dark);
+}
+.two-fields {
+    display: flex;
+    justify-content: space-between;
+}
+.field-container {
+    display:flex;
+    flex-direction: column;
+    
+}
+.half {
+    width:45%;
+}
+.thirty-percent {
+    width:25%;
+}
+.seventy-percent {
+    width:65%;
+}
+.field-container input {
+    width:100%;
+    box-sizing: border-box;
+}
+.spacing {
+    width:10%;
+}
+fieldset {
+    border:none;
+    padding:0;
+    margin: 1rem 0 3rem 0;
+}
+fieldset legend {
+    padding-bottom: 1rem;
+    display: flex;
+    justify-content: left;
+}
+.radio, .terms {
+    position: relative;
+    padding-left: 35px;
+    margin-bottom: 12px;
```

Comment:
Try to avoid px measurements, and use rem or another form of relative measurements where possible.